spec/requests/schools/participants_request_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Schools::Participants", type: :request do
+  let(:user) { create(:user, :induction_coordinator) }
+  let(:school) { user.induction_coordinator_profile.schools.first }
+  # TODO: ?
```

Comment:
Hmmm, what could it mean?

Reply:
lool thanks for reminding me - i started this test file by copying it from another file which was setting up `cohort` literally just to get the `start_year` from it (which is specified here). was trying to figure out if there was a reason `2021` wasn't just hardcoded in the test instead